python/deps/requirements_rtd.txt

```diff
@@ -6,6 +6,6 @@ Pillow
 pyarrow>=2.0
 pycocotools==2.0.2
 pyspark>=3
-pytorch
+torch
```

Comment:
is this because it's pytorch in conda install but torch with pip install?

Reply:
Yes, seems to be so.